core/objects.go

```diff
@@ -220,8 +222,10 @@ type Challenge struct {
 	// used
 	ValidationRecord []ValidationRecord `json:"validationRecord,omitempty"`
 	// The time at which the server validated the challenge. Required by
-	// RFC8555 if status is valid.
-	Validated *time.Time `json:"validated,omitempty"`
+	// RFC8555 if status is valid. This is only used for display to the
+	// client so is set to a string to control display format. Per RFC8555
+	// the display format should be RFC3339.
+	Validated string `json:"validated,omitempty"`
```

Comment:
Despite this being for presentation only, and therefore a string making sense, I think that "uniformity with the surrounding code" should be the higher priority. Our other user-facing timestamps are represented as `time.Time`s, and get converted to RFC3339 formats (as is required by RFC8555 for all of them) on their way out the door just fine.

Reply:
I reworked the entire change, as part of it, this is back to a *time.Time.